unit-tests/src/org/commcare/android/tests/initialization/AppInitializationTest.java

```diff
@@ -0,0 +1,62 @@
+package org.commcare.android.tests.initialization;
+
+import org.commcare.android.CommCareTestRunner;
+import org.commcare.android.database.DbUtil;
+import org.commcare.android.util.TestAppInstaller;
+import org.commcare.android.util.LivePrototypeFactory;
+import org.commcare.dalvik.BuildConfig;
+import org.commcare.dalvik.application.CommCareApplication;
+import org.commcare.suite.model.Profile;
+import org.javarosa.core.reference.ReferenceManager;
+import org.javarosa.core.reference.ResourceReferenceFactory;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.annotation.Config;
+
+/**
+ * Tests that use the ability to install a CommCare app and login as a test
+ * user.
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+@Config(application = org.commcare.dalvik.application.CommCareApplication.class,
+        constants = BuildConfig.class)
+@RunWith(CommCareTestRunner.class)
```

Comment:
FWIW: I think we can add the static initialization stuff to this class if we wanted, which would make the test setups much easier.